style.css

```diff
@@ -0,0 +1,473 @@
+  padding-top: 10%;
+  color: #ec5242;
+}
+
+.Meco {
+  font-size: 50px;
+  color: #ec5242;
+  margin-top: 0%;
+  font-weight: 700;
+}
+
+.welcome-2 {
+  margin: 0%;
+  background: -webkit-linear-gradient(#1692c4, #133fcf);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.main-headline p {
+  padding-top: 16%;
+  background: -webkit-linear-gradient(#0a445c, #8668da);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  width: 36%;
+}
+
+.bottom {
+  font-size: 14px;
+  box-sizing: border-box;
+  background-color: #f8f8f8;
+  opacity: 92%;
+  padding: 1%;
+  padding-bottom: -15%;
+  border: 10px solid white;
+  margin-left: -4%;
+}
+
+.date {
+  font-weight: 700;
+  color: #595959;
+}
+
+.venue {
+  color: #868686;
+}
+
+.menu_items {
+  display: none;
+}
+
+.program {
+  position: relative;
+  top: 67%;
+  width: fit-content;
+  background-image: url(./Images/repeat-bg.png);
+  padding: 24px;
+  left: 0%;
+}
+
+.prog-items {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 20px;
+  color: white;
+  padding: 20px;
+  margin-bottom: 7%;
+  background: rgba(255, 255, 255, 0.1);
+}
+
+.prog-items h2 {
+  color: #ec5242;
+}
+
+.prog-items img {
+  width: 21%;
+}
+
+.red-bar,
+.red-bar_1 {
+  width: 53px;
+  height: 16px;
+  border-bottom: 1px solid rgb(239, 81, 60);
+  margin: 0 auto 54px;
+}
+
+.red-bar_1 {
+  margin-top: -7%;
+}
+
+.section-heading {
+  font-weight: bold;
+  text-align: center;
+  color: white;
+}
+
+.section-heading_1 {
+  color: black;
+}
+
+.register-btn {
+  width: 50%;
+  height: 50px;
+  background-color: #ec5242;
+  border-radius: 4px;
+  border-color: transparent;
+  color: #fff;
+  font-size: 1.1rem;
+  cursor: pointer;
+  align-self: center;
+  margin: 30px 5px 35px 75px;
+}
+
+.speaker_section {
+  position: relative;
+  top: 242%;
+}
+
+.partner-img {
+  width: 120px;
+  height: 120px;
+}
+
+.grp-2 h2 {
+  height: 24px;
+  width: 197px;
+  margin-top: 2%;
+  margin-bottom: 2%;
+}
+
+#speaker_section > h2 {
+  text-align: center;
+  margin-bottom: 10%;
+}
+
+#speaker_section > hr {
+  margin: 10px auto 54px;
+  width: 60px;
+  border: 1px solid #ec5242;
+}
+
+#speaker_section-list {
+  display: flex;
+  flex-direction: column;
+  gap: 48px;
+  padding: 16px;
+}
+
+.grp-1 {
+  gap: 20px;
+  display: flex;
+}
+
+.grp-2 h4 {
+  margin-top: 2%;
+  color: red;
+  margin-bottom: 0%;
+  font-style: italic;
+}
+
+.grp-2 p {
+  margin-top: 7%;
+  margin-bottom: 20%;
+}
+
+#whole-program,
+.partners-section,
+.footer-rights {
+  display: none;
+}
+
+.aside-section {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
+  #ham-menu {
+    display: none;
+  }
+
+  .main-headline {
+    height: 1076px;
+    width: 99%;
+    background-image: url(./Images/mainbg.png);
+    background-size: 124% 93%;
+    background-repeat: no-repeat;
+    background-color: #f1f1f1;
+
+    /* padding-left: 2%; */
+    margin-left: -1%;
```

Comment:
- [ ] I have noticed that you are applied the changes required by the previous reviewer and I congratulate you upon that 👏. However, there is a new error that was introduced as you were applying the changes. The `.main-headline` height is too big and therefore the design does not match the original design. Kindly consider removing the height or use desired styling to ensure that you are matching the original design ☺️.
- [ ] [OPTIONAL] Please consider removing the width, as now it has no effect on the design of the application. Additionally kindly consider removing the commented code to avoid messy code and make your code easier to maintain.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/233766176-987d73c1-1b70-4cf6-89cf-1c5ca7f6e6bf.png)

Design at previous reviewer stage
![prev design](https://user-images.githubusercontent.com/99394374/233766207-a8d0f0c1-4cd8-4ef2-a754-0f867dbd5e29.png)